setup.py

```diff
@@ -27,8 +27,8 @@
 ase_deps = ["ase~=3.19"]
 cif_deps = ["numpy~=1.19"]
 pdb_deps = cif_deps
-pymatgen_deps = ["pymatgen~=2020.6"]
-jarvis_deps = ["jarvis-tools~=2020.6"]
+pymatgen_deps = ["pymatgen~=2020.7"]
+jarvis_deps = ["jarvis-tools~=2020.7"]
```

Comment:
Do you think we should do anything special with non-semver packages here? `~=2020.7` will fairly arbitrarily restrict users to using releases from 2020 only... Can obviously decide it outside of this PR, but I would think maybe just `>=2020` (or even earlier) would be okay for now?

Maybe @shyamd has thoughts on this too...

Reply:
That makes sense for CalVer packages, yeah. Should we do it in this PR already?

Reply:
Up to you! Can re-accept straight away.